### DISCUSSION TITLE
Adding images to custom model documents

### DISCUSSION THREAD

**Discussion by Genai-labs** [open-webui/open-webui#5146] [Ideas]
Currently, the creation of personalized agents/models based on a document set only allows for the upload of text documents, but does not support the inclusion of images. I assume this limitation is due to the document extraction method used. However, it could be beneficial to enable image uploads in the future, particularly for cases where the content of the images—such as diagrams or visual data—could provide added value to the model’s performance and comprehension.

**Comment by coder3337** [comment]
also interested in uploading images to document library. Also, how do we access the uploads folder from outside docker? for eg to remove old files
